cloud_playbook.yml

```diff
@@ -18,3 +18,26 @@
      - zimon/node
      - zimon/cluster
      - zimon/postcheck
+
+  # Below task specific to cloud deployment.
+  tasks:
+    - block:
+       - name: accept client lisence for compute descriptor node
+         command: mmchnode --client -N "computedescnodegrp"
```

Comment:
@rajan-mis Does it make sense to add the full path to mmchnode (usr/lpp/mmfs/bin) since this directory may not have been setup in the path.